contracts/src/v0.8/tests/VerifiableLoadLogTriggerUpkeep.sol

```diff
@@ -12,8 +12,26 @@ contract VerifiableLoadLogTriggerUpkeep is VerifiableLoadBase, FeedLookupCompati
   ];
   string public feedParamKey = "feedIdHex";
   string public timeParamKey = "blockNumber";
+  bool public autoLog;
+  bool public useMercury;
```

Comment:
@FelixFan1992 where i this being used? I couldn't find it. If you can actually add this feature then it would be really good.

Reply:
updated